test/integration/scheduler/scheduler_test.go

```diff
@@ -433,7 +422,7 @@ var _ = ginkgo.Describe("Scheduler", func() {
 		}, framework.Timeout, framework.Interval).Should(gomega.Equal(pointer.Bool(false)))
 		gomega.Expect(createdJob1.Spec.Template.Spec.NodeSelector[instanceKey]).Should(gomega.Equal(onDemandFlavor.Name))
 
-		job2 := testing.MakeJob("on-demand-job2", ns.Name).Queue(devBEQueue.Name).Request(corev1.ResourceCPU, "8").Obj()
+		job2 := testing.MakeJob("on-demand-job2", ns.Name).Queue(prodQueue.Name).Request(corev1.ResourceCPU, "5").Obj()
```

Comment:
just to make sure I am understanding the test correctly, the change from 8 to 5 is not necessary, right?

Reply:
because it reuse `prodClusterQ` again, min of it in `onDemandFlavor` is `5`.
```go
prodClusterQ = testing.MakeClusterQueue("prod-cq").
	Cohort("prod").
	Resource(testing.MakeResource(corev1.ResourceCPU).
		Flavor(testing.MakeFlavor(spotTaintedFlavor.Name, "5").Max("5").Obj()).
		Flavor(testing.MakeFlavor(onDemandFlavor.Name, "5").Obj()).
		Obj()).
	Obj()
```

Reply:
but the min was also 5 for the dev BE:

```
devBEClusterQ = testing.MakeClusterQueue("dev-be-cq").
			Cohort("be").QueueingStrategy(kueue.BestEffortFIFO).
			Resource(testing.MakeResource(corev1.ResourceCPU).
				Flavor(testing.MakeFlavor(onDemandFlavor.Name, "5").Max("15").Obj()).
				Obj()).
			Obj()
```

Reply:
`clusterQ` created in this case share the same `Cohort` with `prodClusterQ`, so we change to use `prodClusterQ`. But I find it's also ok to keep `8` here instead of changing to `5`, I revert it.
```go
clusterQ := testing.MakeClusterQueue("cq").
	Cohort("prod").
	Resource(testing.MakeResource(corev1.ResourceCPU).
		Flavor(testing.MakeFlavor(onDemandFlavor.Name,
			"5").Obj()).
		Obj()).
	Obj()
```